### ISSUE TITLE
Describe loss function + optimizer in Hyperparameters

### ISSUE THREAD

**Issue by petermcaughan** [petermcaughan/petermcaughan.github.io#5]

**Comment by petermcaughan** [comment]
Learning rate / learning rate decay?
